ingest-app/src/cmr/ingest/validation/additional_attribute_validation.clj

```diff
@@ -3,6 +3,7 @@
   (:require
    [clojure.string :as str]
    [cmr.common.util :as util]
+   [clojure.string :refer [lower-case]]
    [cmr.umm-spec.additional-attribute :as aa]))
 
 (defn- aa-range-reduced?
```

Comment:
You might want to add some unit tests for these utility functions to make sure the lower-case stuff works the way you expect.